services/collections-publisher/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ruby:2.6.3
+
+RUN apt-get update -qq && apt-get upgrade -y && apt-get install -y vim
+RUN apt-get install -y libxss1 libappindicator1 libindicator7
+RUN wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb && apt install -y ./google-chrome*.deb
+
+RUN curl -sL https://deb.nodesource.com/setup_12.x | bash -
```

Comment:
I think we’re missing the actual install for ‘nodejs’ now. You might want to try ‘govuk-docker build collections-publisher-lite’ to ensure you’re working with the image represented by this file, as I suspect the tests will now fail if nodejs is required.

Reply:
Running ‘govuk-docker build collections-publisher-lite’ gave me an argument error. 

But I ran `govuk-docker build` which did
```
/Users/jessicajones/govuk/govuk-docker/bin/govuk-docker compose run collections-publisher-lite sh -c 'rake db:migrate 2>/dev/null || rake db:setup'
docker-compose -f [...] run collections-publisher-lite sh -c rake db:migrate 2>/dev/null || rake db:setup
```
...and the tests are still green and the app seems to be running fine in the browser. Guess nodejs isn't required?

Reply:
I’m afraid not. The image still needs to be rebuilt. The build command should probably be changed to actually try and build the image - it wasn’t necessary in the beginning because it gets built automatically the first time.

Reply:
`govuk-docker compose down` and then `govuk-docker build`? Or do I need to run Leena's trifecta of destruction explode all the things commands?

Reply:
Oops - I gave you the wrong command earlier. I meant: `govuk-docker compose build collections-publisher-lite`.

This command should really be in all Makefiles to ensure that the image for each service is in-sync with its Dockerfile - would you be willing to do a PR on that?

Reply:
ah ok that works, and now I'm getting the node errors. Will add it back 👍